modules/sc-mesh-secure-deployment/src/nats/mdm_agent.py

```diff
@@ -66,6 +66,7 @@ def __init__(
         self.__debug_config_interval: int = Constants.FAIL_POLLING_TIME_SECONDS.value  # TODO: possible to update from MDM server?
 
         self.__url: str = "defaultmdm.local:5000"  # mDNS callback updates this one
+        self.__fallback_url: str = "[fdbb::1]:5000"
```

Comment:
Not sure did I understand Seb's comment correctly but should this be then:
self.__fallback_url: str = f"[{self.__IPV6_WHITE_PREFIX}::1]:5000"

Reply:
I think so, will clarify now :+1: